packages/valory/skills/abstract_round_abci/tests/test_behaviours_utils.py

```diff
@@ -1574,8 +1575,8 @@ def test_is_timeout_expired(
     @pytest.mark.parametrize("default", (True, False))
     @given(
         st.datetimes(
-            min_value=datetime(2, 1, 1),
-            max_value=datetime(9999, 1, 1),
+            min_value=datetime(1970, 1, 1, 0, 0, 0, tzinfo=timezone.utc),
+            max_value=datetime(9999, 1, 1, 0, 0, 0, tzinfo=timezone.utc),
```

Comment:
change introduced because of this error observed on Windows CI runs: https://github.com/valory-xyz/open-autonomy/actions/runs/3306777482/jobs/5457971302

```
___ TestBaseBehaviour.test_wait_until_transaction_delivered_raises_timeout ____

self = <packages.valory.skills.abstract_round_abci.tests.test_behaviours_utils.TestBaseBehaviour object at 0x00000230018B4370>
_ = ()
gen = <generator object BaseBehaviour._wait_until_transaction_delivered at 0x0000023002D8DA10>

    def test_wait_until_transaction_delivered_raises_timeout(self, *_: Any) -> None:
        """Test '_wait_until_transaction_delivered' method."""
        gen = self.behaviour._wait_until_transaction_delivered(MagicMock(), timeout=0.0)
>       with pytest.raises(TimeoutException):
E       Failed: DID NOT RAISE <class 'packages.valory.skills.abstract_round_abci.behaviour_utils.TimeoutException'>

tests\test_behaviours_utils.py:1265: Failed
---------------------------- Captured stdout call -----------------------------
error: Invalid type for content 'url'. Expected 'str'. Found '<class 'unittest.mock.MagicMock'>'.

---------------------------- Captured stderr call -----------------------------
Error: 0-23 11:10:12,947][ERROR] Invalid type for content 'url'. Expected 'str'. Found '<class 'unittest.mock.MagicMock'>'.

------------------------------ Captured log call ------------------------------
ERROR    aea.packages.valory.protocols.http.message:message.py:294 Invalid type for content 'url'. Expected 'str'. Found '<class 'unittest.mock.MagicMock'>'.
```